tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,44 @@
+## Use cases
+
+### Sentence classification
+
+For sentence classification, we’re only only interested in BERT’s output for the [CLS] token, so we get the sentence embedding using
```

Comment:
Can you explain this simpler? You can see other task pages and the contribution guidelines for the technical writing we look for 🙂